Fix build with newer glibc versions

This patch has been backported from upstream:

http://cgit.freedesktop.org/udisks/commit/?h=udisks1&id=9829152b12a8924d2e091a00133ed1a3a7ba75c0

Signed-off-by: Vicente Olivert Riera <Vincent.Riera@imgtec.com>

From 9829152b12a8924d2e091a00133ed1a3a7ba75c0 Mon Sep 17 00:00:00 2001
From: Alexandre Rostovtsev <tetromino@gentoo.org>
Date: Fri, 29 May 2015 21:09:39 -0400
Subject: fix build with newer glibc versions

https://bugs.freedesktop.org/show_bug.cgi?id=90778

diff --git a/src/helpers/job-drive-detach.c b/src/helpers/job-drive-detach.c
index eeafcab..d122a1f 100644
--- a/src/helpers/job-drive-detach.c
+++ b/src/helpers/job-drive-detach.c
@@ -18,6 +18,7 @@
  *
  */
 
+#include <sys/stat.h>
 #include <stdio.h>
 #include <string.h>
 #include <errno.h>
-- 
cgit v0.10.2
